src/components/molecules/SpaceEditForm/SpaceEditForm.scss

```
@@ -9,7 +9,12 @@
   &__portal {
     display: flex;
     flex-direction: column;
-    padding: $spacing--xl;
+
+    .AdminInput,
+    .AdminTextarea,
+    .AdminSection {
+      padding: 0px $spacing--xl 0px $spacing--xl;
```

```suggestion
      padding: 0 $spacing--xl;
```

From https://developer.mozilla.org/en-US/docs/Web/CSS/padding :
2. When two values are specified, the first padding applies to the top and bottom, the second to the left and right.
3. When three values are specified, the first padding applies to the top, the second to the right and left, the third to the bottom.